pkg/helper/versions/helpers_test.go

```diff
@@ -72,7 +71,18 @@ var _ = Describe("Version Helpers", Ordered, func() {
 				"4.12.0-0.a",
 			),
 		)
-
 	})
+})
 
+var _ = Describe("Validates Format Major Minor Patch", func() {
+	DescribeTable("",
```

Comment:
I think even though it's the only block in this `Describe` block, we should have a title/description for the table block